public/sass/pages/_explore.scss

```diff
@@ -148,10 +183,29 @@
   }
 }
 
-@media only screen and (max-width: 702px) {
-  .explore-toolbar-content-item:first-child {
-    padding-left: 2px;
-    margin-right: 0;
+@media only screen and (max-width: 810px) {
+  .explore-toolbar {
+    .explore-toolbar-content-item {
+      &:first-child {
+        padding-left: 2px;
+        margin-right: 0;
+      }
+    }
+  }
+}
+
+@media only screen and (max-width: 634px) {
+  .datasource-picker {
+    .ds-picker {
+      min-width: 50px;
+      max-width: 50px;
+
+      .gf-form-select-box__single-value {
```

Comment:
This is an internal class, style defined in _Select.scss  , overriding it from the outside like this is not ok.

Reply:
Moved to _Select.scss and refactored it to be a prop.